owncloudApp/src/main/java/com/owncloud/android/MainApp.kt

```diff
@@ -84,6 +85,9 @@ class MainApp : Application() {
         // initialise thumbnails cache on background thread
         ThumbnailsCacheManager.InitDiskCacheTask().execute()
 
+        // check for dangling account data and delete it
+        FileStorageUtils.performCleanup(this)
```

Comment:
Maybe trigger this each time the app is opened is too much. Maybe triggering it once a day or something like that would be enough. We could do this on the background using a worker as this PR: https://github.com/owncloud/android/pull/3337

BTW, I think that we could find a better name for this function. With a more specific name, the comment could be omitted 👀

Reply:
I don't know about triggering a cleanup every day. This seems a bit to complicated to me. Especially because the check is not really computational expensive.